app/controllers/api/projects.php

```diff
@@ -325,13 +325,13 @@
 
             $usage = new Document([
                 'range' => $range,
-                'requests' => $stats['requests'],
-                'network' => $stats['network'],
-                'functions' => $stats['executions'],
-                'documents' => $stats['databases.documents.count'],
-                'collections' => $stats['databases.collections.count'],
-                'users' => $stats['users.count'],
-                'storage' => $stats['storage.total']
+                'requests' => $stats[$metrics[0]],
```

Comment:
We should add null safety operators here.

Reply:
@eldadfux Why? We always fill `$stats` with all the `$metrics` listed even if there's no data.

Reply:
We just need to make sure we fallback to 0 or null whatever works using `?? 0`.

Reply:
`$stats[$metric]` is an array and by default it's always set to `[]` in you look at https://github.com/appwrite/appwrite/blob/246d9b94b7084bfd8af502423a7e1a65444ca995/app/controllers/api/projects.php#L300 is the fallback still necessary?

Reply:
I guess it should work, but don't we prefer to be consistent in case someone might change the above code in the future?

Reply:
updated in all the usage endpoints to handle null value